content/cloud-docs/users-teams-organizations/organizations.mdx

```diff
@@ -13,179 +13,188 @@ description: >-
+#### Authentication
 
-### Policies
+Organization owners can determine when users must reauthenticate and require [two-factor authentication](/cloud-docs/users-teams-organizations/2fa) for all members of the organization.
 
-The policies page is a deprecated interface for managing Sentinel policies. Use the policy sets page instead.
+#### SSH Keys
 
-### Policy Sets
+Manage [SSH keys for cloning Git-based modules](/cloud-docs/workspaces/settings/ssh-keys) during Terraform runs. This does not include keys to access a connected VCS provider.
 
--> **Note:** Sentinel policies are a paid feature, available as part of the **Team & Governance** upgrade package. [Learn more about Terraform Cloud pricing here](https://www.hashicorp.com/products/terraform/pricing).
+#### SSO
 
-The policy sets page is for creating groups of Sentinel policies from a connected VCS repository, and assigning those policy sets to workspaces. It is available to users with permission to manage policies. ([More about permissions.](/cloud-docs/users-teams-organizations/permissions))
+-> **Note:** Single sign-on is a paid feature, available as part of the **Business** plan. Refer to [Terraform pricing](https://www.hashicorp.com/products/terraform/pricing) for details.
 
-[permissions-citation]: #intentionally-unused---keep-for-maintainers
+Organization owners can [set up an SSO provider](/cloud-docs/users-teams-organizations/single-sign-on) for the organization.
 
-Sentinel is an embedded policy-as-code framework that can enforce rules about Terraform runs within an organization. See [Sentinel](/cloud-docs/sentinel) for more information about Sentinel, or [Managing Sentinel Policies](/cloud-docs/sentinel/manage-policies) for details about these two settings pages.
+### Version Control
 
-### Run Tasks
+#### General
 
-Run tasks let you integrate third-party tools and services at specific stages in the Terraform Cloud run lifecycle. The Run Tasks page is for managing the tasks which are available for workspaces to consume.
+Workspaces that use part of a shared repository do not typically run plans for changes that do not affect their files. This includes [speculative plans](/cloud-docs/run/remote-operations#speculative-plans) on pull requests. Since **pending** status checks can block pull requests, workspaces automatically send passing commit statuses for any PRs that do not affect their files.
 
-Refer to [Run Tasks](/cloud-docs/workspaces/settings/run-tasks) for more information.
+You can disable this behavior if it creates too many status checks to your VCS provider. You may want to do this if you have a large number of workspaces sharing one VCS repository.
 
-### VCS Providers
+#### VCS Events
 
-The VCS providers page is used for configuring VCS providers for use within the organization. It is available to users with permission to manage VCS settings. ([More about permissions.](/cloud-docs/users-teams-organizations/permissions))
+-> **Note:** This feature is in beta.
```

Comment:
```suggestion
-> **Note:** This feature is in beta as of May 2022.
```
On features like this, is it worth mentioning a timestamp of some kind to bookmark when the docs were made, or is the feature's release status being tracked in some other channel?

Reply:
this is an interesting question. We don't typically put times into the docs like this if we can help it. Theoretically, the docs should be up-to-date. BUT thank you for flagging this because I actually do want to flag whether this is still in beta.....there are some inconsistencies in the UI